lib/start.js

```diff
@@ -16,7 +16,7 @@
 
 'use strict'
 
-process.env.DEBUG = `sdk:usbboot,${process.env.DEBUG}`
+process.env.DEBUG = `sdk:usbboot${process.env.DEBUG ? `,${process.env.DEBUG}` : ''}`
```

Comment:
I wonder if this version is any more readable?
```js
process.env.DEBUG += `${process.env.DEBUG ? `,` : ''}sdk:usbboot`
```
(it feels more logical to me :wink: )

Reply:
Nope, ignore me, looks like `+=` stringifies `null` into `'null'` :(

Reply:
This doesn't work if `process.env.DEBUG` is `undefined` unfortunately.

Reply:
Yeah, which is what I was showing with my code-snippet ;-)

Reply:
Yeah Github seems to like dynamically loading the first comment to trick the user (me) into a false sense of security but then subsequently load no more comments or edits

Reply:
Can we please not muck with the `process.env.DEBUG` here, and enable certain scopes we definitely want to be on all the time in their respective modules? For example, to have `sdk:usbboot` permamently enabled we can do the following in `lib/sdk/usbboot/index.js`:

```diff
 const debug = require('debug')('sdk:usbboot')
 const usb = require('./usb')
 const protocol = require('./protocol')

+debug.enabled = true
+
 /**
  * @summary The name of this adaptor
  * @public
```

Reply:
Can we please not muck with the `process.env.DEBUG` here, and enable certain scopes we definitely want to be on all the time in their respective modules? For example, to have `sdk:usbboot` permamently enabled we can do the following in `lib/sdk/usbboot/index.js`:

```diff
 const debug = require('debug')('sdk:usbboot')
 const usb = require('./usb')
 const protocol = require('./protocol')

+debug.enabled = true
+
 /**
  * @summary The name of this adaptor
  * @public
```

Reply:
Ping @shou @jviotti ^